packages/admin/cms-admin/src/pages/pagesPage/PagesPage.tsx

```diff
@@ -178,17 +179,19 @@ export function PagesPage({
                                 />
                             </ActionToolbarBox>
                             <FullHeightPaper variant="outlined">
-                                {loading && <CircularProgress />}
-
-                                <PageTree
-                                    ref={refPageTree}
-                                    pages={pagesToRenderWithMatches}
-                                    editDialogApi={editDialogApi}
-                                    toggleExpand={toggleExpand}
-                                    onSelectChanged={onSelectChanged}
-                                    category={category}
-                                    siteUrl={siteConfig.url}
-                                />
+                                {loading ? (
+                                    <Loading behavior="fillParent" />
+                                ) : (
```

Comment:
I'm not sure that this change in behavior (hide page tree during loading) is desired: In production, the page tree is polled every 10 seconds. Would this cause the page tree to disappear every 10 seconds? Could you please verify that this is not the case by temporarily enable polling?

Reply:
Thanks for testing!